insertMealData.js

```diff
@@ -0,0 +1,28 @@
+const _ = require('lodash'),
+  csv = require('csvtojson'),
+  pg = require('pg'),
+  connectionString = 'postgres://localhost:5432/jkitchen',
+  psql = new pg.Client(connectionString),
+  csvFilePath='./mealData.csv';
+
+psql.connect();
+const task = 'INSERT INTO "Meals"(title,price,available_quantity,image,description,created_at,updated_at) values($1, $2, $3, $4, $5,CURRENT_TIMESTAMP,CURRENT_TIMESTAMP)'
+
+
+csv()
+.fromFile(csvFilePath)
+.on('csv',(csvRow) => {
+  psql.query(task, csvRow, (err, res) => {
+    if(err) {
+      console.log(err);
```

Comment:
you can use a logger instead of using console statements to debug. Note that console.log statements isn't a good idea.